src/Libraries/2D/Source/Flat8/fl8chfl8.c

```diff
@@ -24,40 +24,41 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * $Log: fl8chfl8.c $
  * Revision 1.4  1993/10/26  02:23:49  kevin
  * Use default clut if passed cl=NULL.
- * 
+ *
  * Revision 1.3  1993/10/19  09:50:17  kaboom
  * Replaced #include <grd.h" with new headers split from grd.h.
- * 
+ *
  * Revision 1.2  1993/10/08  01:15:07  kaboom
  * Changed quotes in #include lines to angle brackets for Watcom.
- * 
+ *
  * Revision 1.1  1993/10/06  14:39:25  kevin
  * Initial revision
  */
 
 #include "cnvdat.h"
-#include "scrmac.h"
 #include "flat8.h"
+#include "scrmac.h"
 
 /* draw an unclipped, horizontally flipped flat 8 bitmap to a flat 8
    canvas through a color lookup table (clut). */
-void flat8_clut_hflip_flat8_ubitmap (grs_bitmap *bm, short x, short y,uchar *cl)
-{
-	short w;                   /* bitmap width */
-	short h;                   /* height */
-	uchar *src;                /* pointer into source bitmap */
-	uchar *dst;                /* pointer into canvas memory */
-	ushort	row;
-	
-	if (cl == NULL) cl=gr_get_clut();
-	h = bm->h;
-	src = bm->bits;
-	dst = grd_bm.bits + y*grd_bm.row + x+bm->w-1;
-	while (h--) {
-	w = bm->w;
-	while (w--)
-	 *dst-- = cl[*src++];
-	src += bm->row-bm->w;
-	dst += grd_bm.row+bm->w;
-	}
+void flat8_clut_hflip_flat8_ubitmap(grs_bitmap *bm, short x, short y,
+                                    uchar *cl) {
```

Comment:
I don't like how short this formatting is wrapping things - what arguments are you running when doing the code cleanup?

Reply:
clang-format with default settings

Reply:
There 3 types of BracketAlignmentStyle https://clang.llvm.org/docs/ClangFormatStyleOptions.html , which would be better?

Reply:
I think the ColumnLimit is what bugs me, 80 seems super narrow and makes me scroll way too much. Looking Glass seemed to not mind wide lines, maybe we can bump up to 120 or 130?

Reply:
Implemented .clang-format style with column-width 120.